packages/core/src/edge/edge-builder.ts

```diff
@@ -198,7 +198,7 @@ export class EdgeBuilder extends FieldBuilder {
   }
 
   protected buildOp<
-    T extends OpBuilders,
+    T extends OpBuilders | RecurseBuilder,
```

Comment:
`buildOp` is just for building operators, it shouldn't be responsible of building anything else.